docs/standards/v1/STANDARDS.md

```diff
@@ -0,0 +1,454 @@
+# Hack Oregon Data APIs v1 Style Guide
+
+## Overview
+
+The intent of this document is to provide some guidance for API developers when building APIs under Hack Oregon Projects. Our aim is to follow general RESTful API best practices, and work with solutions/standards that are built into our technology stack.
+
+We aim to also provide the highest level of interoperability between our APIs. By adopting standards between teams, we can better support users connecting to multiple APIs.
+
+### API Technologies
+* **HTTPS:** All API requests must be made over HTTPS.
+* **JSON:** All API request and response bodies will be JSON.
```

Comment:
From the examples, it seems we're following JSON:API. Can we state that explicitly and link to [https://jsonapi.org/format/](https://jsonapi.org/format/)?